docs/integrations/sources/sftp-bulk.md

```diff
@@ -113,6 +113,14 @@ For example, assuming your folder path is not set in the connector configuration
 
 If your files are in a folder, include the folder in your glob pattern, like `my_folder/my_prefix_*.csv`.
 
+#### Copy Raw Files Configuration
```

Comment:
I don't remember the syntax, but I recall there's a way to make this section of the docs take focus when the associated control is selected on the left-hand config.

Reply:
Do you remember any doc with this behavior?